bin/alluxio

```diff
@@ -11,7 +11,7 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-debug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
```

Comment:
Yeah. I add a flag description block before the COMMAND description, and update the document content of debugging-guide.md.